src/components/form-control/form-control.jsx

```diff
@@ -21,13 +22,18 @@ function isInvalid(error, readOnly) {
 export default function FormControl(props) {
   return (
     <div className={props.className} onKeyDown={props.onKeyDown} role="presentation">
-      <label
-        className={getLabelClassName(props.error, props.readOnly)}
-        htmlFor={props.id}
-        id={props.labelId}
-      >
-        {props.label}
-      </label>
+      <div className={styles['label-wrapper']}>
```

Comment:
This is a long shot buuuut bear with me:

What if instead of using intermediary `<div>` for layout purposes.... the container div is designated as a `display: grid` and we use a template?

```css
.container {
  display: grid;
  grid-template-areas:
    label help
    input input
    valid valid;
```

Reply:
I started to go down this route but doesn't this mean that each child of `FormControl` needs to have `grid-area: input`? I'm new to grid so I could be misunderstanding but it seems like that'd be a bit more invasive than the extra div

Reply:
Right... 🤔  Yeah, let's keep it as-is!